nf_core/modules/module_test.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+"""
+The ModulesTest class runs the tests locally
+"""
+
+import logging
+import questionary
+import os
+import pytest
+import sys
+import rich
+import subprocess
+import shlex
+from pathlib import Path
+from shutil import which
+
+import nf_core.utils
+from .modules_repo import ModulesRepo
+
+log = logging.getLogger(__name__)
+
+
+class ModulesTest(object):
+    """
+    Class to run module pytests.
+
+    ...
+
+    Attributes
```

Comment:
If I'm being picky, these are for the `__init__()` def really, so you could probably move them there. Unless this is some kind of Pythonic way of documenting stuff that I'm not aware of. Low prio though.